### ISSUE TITLE
Added some people to Owner group

### ISSUE THREAD

**Issue by bvssvni** [PistonDevelopers/piston#888]
There are a few cases where somebody needs Owner permission, for example to add people to the Admin group. I have done this task up to this day, but I would like to have some reserves in case I am not available. I added some people to the group before I did the analysis, but there are other things we could do to organize it better so don't put too much weight on being added to the group. The difference between the Admin group and Owner group is pretty small in terms of permissions.

The risk for part of the Piston project stopping due to inavailability is "k * p^n" where "p" is the average probability of inavailability and "k" is the chance an issue appears that require special permission. The estimates are based on previous year for 3 people 20/365*(10/365)^3 gives less than 0.0002% per day.

There is a risk of somebody with malicious intends getting access to an Owner account is "1 - (1 - p)^n" where "p" is the average probability per account accident of size "n". Assuming "p" is 1/10000 per day, it gives 0.03% which is significantly higher and might have more serious consequences.

It seems that the optimal size of the Owner group is somewhere around 3-5 with our current situation. This might change over time, but we do not need a perfect solution, it might be a better idea to add secondary measures in case something goes wrong.

**Comment by sinistersnare** [comment]
I think it is important to have owners in different timezones. You are in europe as I understand it (you are usually not on when I am :P ), so it may be good to have an owner in one of the US timezones.

**Comment by bvssvni** [comment]
OK, there is not that much need for many owners since we have admins in all the time zones. I added @nwin and @mitchmindtree as owners which have been active since the beginning.

I estimate the tasks of the owner team to be about 20 times x 1 min a year with no hard time limit. If I for some reason can't do these tasks, then the other owners can do it or appoint somebody else. When selecting some other owner, pick people who have been active for several months (longer is better). The owner team has no special role besides keeping the Piston project running through these tasks. All other problems must be opened as issues.

Closing.
